data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/ConditionalExpressionEvaluatorIT.java

```diff
@@ -183,7 +193,8 @@ private static Stream<Arguments> invalidExpressionArguments() {
                 Arguments.of("not null", event("{}")),
                 Arguments.of("not/status_code", event("{\"status_code\": 200}")),
                 Arguments.of("trueand/status_code", event("{\"status_code\": 200}")),
-                Arguments.of("trueor/status_code", event("{\"status_code\": 200}"))
+                Arguments.of("trueor/status_code", event("{\"status_code\": 200}")),
+                Arguments.of("length(\""+testString+") == "+testStringLength, event("{\"response\": \""+testString+"\"}"), true)
```

Comment:
The test which uses this list of args takes two arguments. You have three on this line.

Reply:
Fixed it. Thanks!